docs/documentation/_data/supported_versions.yml

```
@@ -21,7 +21,7 @@ osDistributions:
   ubuntu:
     name: Ubuntu
     url: https://ubuntu.com/download/server
-  '_redos':
+  redos:
```

```suggestion
  '_redos'::
```
It was a dirty trick to proper items sorting  :)

---

@borg-z JFI

---

I see, I thought it was some kind of typo.